core/templates/dev/head/pages/admin/roles_tab/roles_tab_directive.html

```diff
@@ -0,0 +1,100 @@
+<style>
+    .form-horizontal .control-label {
+        text-align: left;
+    }
+</style>
+
+<div class="container-fluid">
+  <div class="row">
+    <br>
+    <h3 class="text-center text-danger">NOTE: Roles can't be modified from here yet.</h3>
```

Comment:
I think you can remove this, so it looks prettier. The admin can't do anything anyway.